Formula/cod.rb

```diff
@@ -0,0 +1,18 @@
+class Cod < Formula
+  desc "Completion daemon for bash/fish/zsh"
+  homepage "https://github.com/dim-an/cod"
+  url "https://github.com/dim-an/cod/archive/9096c4bd0b4a34121f35c323e174a139c30aa8f6.tar.gz"
+  sha256 "dfc8de77d33d3a4e4af46eda3b61e4dbbeddf099d8f85bfc0b4c4a2d693b425c"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "cod"
```

Comment:
Is there no `make install`?

Reply:
There is, but it installs the binary into a specific bin folder on the machine. `build` simply compiles the binary.